homeassistant/components/philips_js/light.py

```diff
@@ -152,7 +151,7 @@ def __init__(
         self._last_selected_effect: AmbilightEffect = None
         super().__init__(coordinator)
 
-        self._attr_supported_color_modes = [COLOR_MODE_HS, COLOR_MODE_ONOFF]
+        self._attr_supported_color_modes = {ColorMode.HS}
```

Comment:
This change was wrong. It does support the OnOff mode too in some cases.

Reply:
It was reverted before merge.